client/components/App.jsx

```diff
@@ -28,24 +31,38 @@ class App extends React.Component {
           }
         })
         this.setState({
-          poster: image
+          poster: image,
```

Comment:
You've probably checked this out but just to be thorough, is the comma necessary to update the state of a particular property?